content/updates/terraform-use-cases.md

```diff
@@ -0,0 +1,202 @@
+
+For AWS users, AWS offers IAM Identity Center (formerly AWS SSO), which simplifies AWS account and role management. But for organizations that haven't jumped on the SSO bandwagon yet, or are using a mix of services with and without SSO support, TF can standardize onboarding and offboarding regardless of vendor. By defining users and roles in a `team.yaml` file, TF Root Modules can automatically create and manage these entities across all the different platforms you're using.
+
+Here’s an excerpt of a `team.yaml` file that describes a hypothetical DevOps team:
+
+`devops_team:`
+
+```yaml
+  name: DevOps
+  description: Internal DevOps Team
+  privacy: closed
+  members:
+    - name: Jane Doe
+      gh_username: JaneyDoe100
+      email: doe@abccorp.com
+      gh_role: maintainer
+      datadog_role: Standard
+    - name: John Smith
+      gh_username: CloudWizard1212
+      email: smith@abccorp.com
+      gh_role: member
+      datadog_role: Read-Only
+    - name: Finn Mertens
+      gh_username: IceKing99
+      email: mertens@abccorp.com
+      gh_role: member
+      datadog_role: Standard
+```
+
+We can use this central file to manage our team in code. Rather than requiring N clicks, adding a new service or user account only requires updating this file.Then, Using TF, we can read this file and deploy this team to all the services. In this example, we’ll update GitHub and Datadog:
+
+```hcl
+locals {
+  # We choose to make this a YAML file we load instead of a variable so that
+  # other team members can easily add / edit / remove entries without any TF
+  # knowledge
+  team_data = yamldecode(file("${path.root}/team.yaml"))
+}
+
+resource "github_team" "devops" {
+  name        = local.team_data.devops_team.name
+  description = local.team_data.devops_team.description
+  privacy     = local.team_data.devops_team.privacy
+}
+
+resource "github_team_members" "devops_members" {
+  for_each = { for member in local.team_data.devops_team.members : member.gh_username => member }
+
+  team_id  = github_team.devops.id
+  username = each.value.gh_username
+  role     = each.value.gh_role
+}
+
+module "datadog_users" {
+  source  = "masterpointio/datadog/users"
+  version = "X.X.X"
+
+  users = [ for member in local.team_data.devops_team.members: {
+      email    = member.email,
+      name     = member.name,
+      role     = [member.datadog_role],
+      username = member.gh_username
+    }
+  ]
+}
+```
+
+This is a simple example, but it should provide an idea of the possibilities when adopting IaC to manage users and roles. By using IaC, instead of having to log in to each platform individually to add or remove team members, we can now change a single file and have everything update automatically when we deploy code changes. Additionally, since all changes are tracked in git, we have a complete history of who made the change, what was changed, when it was changed, and why it was changed.
```

Comment:
```suggestion
This is a simple example, but it should provide an idea of the possibilities when adopting IaC to manage users and roles. By using IaC, instead of having to log in to each platform individually to add or remove team members, we can now change a single file and have everything update automatically when we deploy code changes. Additionally, since all changes are tracked in Git, we have a complete history of who made the change, what was changed, when it was changed, and why it was changed.
```